saga/saga_generators.go

```diff
@@ -146,10 +149,10 @@ func GenId() gopter.Gen {
 	}
 }
 
-// Generator for a Valid Saga State
-func GenSagaState() gopter.Gen {
+// Generator for a Valid Saga State, with a job def if includeJob
+func GenSagaState(includeJob bool) gopter.Gen {
```

Comment:
Is there a reason we wouldn't want the job?  Why not just always have it add it, then the test is more realistic